### ISSUE TITLE
Web interface seeding colors should communicate the torrent's state better with shades of a color

### ISSUE THREAD

**Issue by TorsteinVin** [transmission/transmission#4303]
### What is the issue?

First of all, what a huge upgrade to the web interface, my gosh!! Good job, I'm in love with it! Here is a suggestion to make it easier to know which state a torrent is in with a quick glance.

I think the way the macOS client handles it is brilliant. 

When a download has started, the progress bar is empty, and fills up with blue as it downloads. Once finished downloading, it switches to a light green to indicate it is done, and have started seeding. As seeding progresses the light green is replaced with a dark green progressing and replacing the light green as it is reaching the seed limit  (ie the default 2.0). Once the torrent has stopped seeding, it should switch to a dark gray. 

TL;DR: Web interface seeding color should be a light green indicating finished download that gradually shifts to dark green as it is seeding and reaching seed limit. 

### Which application of Transmission?

Web client

### Which version of Transmission?

4.00 beta 2

**Comment by zdzichu** [comment]
I strongly support using dark gray for torrents with finished seeding. Right now they're indistinguishable from 0% progress (not yet downloaded) torrents.
